-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 2, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 1.304 s <<< FAILURE! - in TestSuite
Oppo_F17.Test9F.SetUp  Time elapsed: 1.249 s  <<< FAILURE!
org.openqa.selenium.SessionNotCreatedException: 
Unable to create a new remote session. Please check the server log for more details. Original error: An unknown server-side error occurred while processing the command. Original error: Device 3e301a7 was not in the list of connected devices
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: '5CG2148T4G', ip: '10.113.85.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Device 3e301a7 was not in the list of connected devices
    at getResponseForW3CError (C:\Users\7340193\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\7340193\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: '5CG2148T4G', ip: '10.113.85.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: driver.version: AndroidDriver
	at Oppo_F17.Test9F.SetUp(Test9F.java:54)
Caused by: java.lang.reflect.InvocationTargetException
	at Oppo_F17.Test9F.SetUp(Test9F.java:54)
Caused by: org.openqa.selenium.WebDriverException: 
An unknown server-side error occurred while processing the command. Original error: Device 3e301a7 was not in the list of connected devices
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: '5CG2148T4G', ip: '10.113.85.6', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_202'
Driver info: driver.version: AndroidDriver
remote stacktrace: UnknownError: An unknown server-side error occurred while processing the command. Original error: Device 3e301a7 was not in the list of connected devices
    at getResponseForW3CError (C:\Users\7340193\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\errors.js:804:9)
    at asyncHandler (C:\Users\7340193\AppData\Local\Programs\Appium Server GUI\resources\app\node_modules\appium\node_modules\appium-base-driver\lib\protocol\protocol.js:380:37)
	at Oppo_F17.Test9F.SetUp(Test9F.java:54)

